easyrsa3/easyrsa

```diff
@@ -782,6 +782,7 @@ $EASYRSA_TEMP_EXT"
 	notice "\
 Certificate created at: $crt_out
 "
+	rm -f "$req_in"
```

Comment:
REQs can be reused at a later date to generate a new certificate.  This is a behavior which may cause problems for users.  I'd rather see this as a command line option or config file variable.